program-runtime/src/prioritization_fee.rs

```diff
@@ -5,6 +5,7 @@ type MicroLamports = u128;
 
 pub enum PrioritizationFeeType {
     ComputeUnitPrice(u64),
+    /// TODO: remove 'Deprecated' after feature remove_deprecated_request_unit_ix::id() is activated
```

Comment:
```suggestion
    // TODO: remove 'Deprecated' after feature remove_deprecated_request_unit_ix::id() is activated
```
This shouldn't be a doc comment

Reply:
This comment hasn't been updated yet